pkg/tsdb/cloudmonitoring/cloudmonitoring.go

```diff
@@ -481,7 +481,11 @@ func calculateAlignmentPeriod(alignmentPeriod string, intervalMs int64, duration
 func formatLegendKeys(metricType string, defaultMetricName string, labels map[string]string,
 	additionalLabels map[string]string, query cloudMonitoringQueryExecutor) string {
 	if query.getAliasBy() == "" {
-		return defaultMetricName
+		if defaultMetricName != "" {
+			return defaultMetricName
+		}
+
+		return metricType
```

Comment:
I'm not sure what else we could use as the label.

Reply:
This is a fine choice 😊